CHANGELOG.md

```diff
@@ -3,6 +3,9 @@ AppEngine version, listed here to ease deployment and troubleshooting.
 
 ## Next Release (replace with git tag when deployed)
 
+ * Update project to use split health checks, run:
+   `gcloud app update --split-health-checks --project dartlang-pub`
```

Comment:
Won't we need to also update the other services to use the new health checks?

Reply:
I've tested that on staging, and they seem to be fine, if the split checks are not defined, appengine  seems to use the default health checks.

Reply:
Okay, that's fine we can finish https://github.com/dart-lang/pub-dev/issues/2111 later :)

Reply:
ah, ack